README.md

```diff
@@ -71,3 +71,17 @@ _Note:_ Acceptance tests create real resources, and often cost money to run.
 ```
 $ make testacc
 ```
+
+You may save your acceptance tests environment variables in the `.env` file, for example:
+
+```sh
+$ cat .env
+HCLOUD_TOKEN=YOUR_API_TEST_TOKEN
+TF_ACC=1
```

Comment:
If you run `test ./...` with this change, it will _always_ run the acceptance tests, not only the unit tests. For our kubernetes projects we use build constraints to solve this, but not sure how well this integrates with Goland.